fishtank/src/utils.ts

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { PathLike, promises } from 'fs'
+
+export const writeJson = (path: PathLike, obj: unknown): Promise<void> => {
+  const jsonData = JSON.stringify(obj)
+  return promises.writeFile(path, jsonData)
```

Comment:
Are these utility functions even worth having? If they did validation/error handling seems valuable but this line above could just be `promises.writeFile(path, JSON.stringify(obj))`

Reply:
That's a good point. I guess no, they're not really needed.